lib/greed/score.rb

```diff
@@ -0,0 +1,26 @@
+module Greed
+  module Score
+    def calcScore(dice)
+      raise ArgumentError.new unless dice
+      score = 0
+
+      # Create hash of count of each die
+      counts = dice.inject(Hash.new(0)) {|hash, num| hash[num] += 1; hash}
+
+      # Calculate score
+      counts.each do |die, count|
```

Comment:
Previously you used `inject` to reduce an array into an object, could it be used here too instead of having a `score` variable?